packages/firestore/src/platform_node/grpc_connection.ts

```diff
@@ -125,11 +125,8 @@ export class GrpcConnection implements Connection {
     rpcName: string,
     token: Token | null
   ): UnaryRpc<Req, Resp> | ReadableRpc<Req, Resp> | DuplexRpc<Req, Resp> {
-    // RPC Methods have the first character lower-cased
-    // (e.g. Listen => listen(), BatchGetDocuments => batchGetDocuments()).
-    const rpcMethod = rpcName.charAt(0).toLowerCase() + rpcName.slice(1);
     const stub = this.ensureActiveStub(token);
-    const rpc = stub[rpcMethod];
+    const rpc = stub[rpcName];
```

Comment:
This is surprising that it works.

Reply:
Looking at the `stub` object in the debugger, it has 2 methods for every RPC, supporting either case now.  E.g. `listen` and `Listen`.